### ISSUE TITLE
Prototype level design

### ISSUE THREAD

**Issue by shauncg** [sinksinksink/FlyingDrunks#4]
Knocking together prototype of a basic starting level. 

Consider objects to block LOS (& weapons), rebounding 'pinged' characters, for wall-jumping etc. 

Best research guides on good practice for competitive FPS.

**Comment by sinksinksink** [comment]
This looks useful: https://www.unrealengine.com/blog/introduction-to-level-design

**Comment by shauncg** [comment]
Re. level visual design - see this comment https://github.com/sinksinksink/FlyingDrunks/issues/14#issuecomment-104278171

**Comment by sinksinksink** [comment]
https://github.com/sinksinksink/FlyingDrunks/milestones

Nearly there!

**Comment by shauncg** [comment]
Huzzah! 

I'll be on this shortly. What I may do is rather than try to develop a sophisticated arena is put together something basic that is symmetrical and contains at least one example of every kind of obstacle, geometry etc I've been envisioning. That way we can progress to the next milestone more promptly. 

More possible resources: 
http://www.ongamedesign.net/designing-fps-multiplayer-maps-part-1/
http://uk.ign.com/articles/2013/10/24/the-science-of-multiplayer-mapping
http://www.g4tv.com/thefeed/blog/post/719216/how-to-build-the-best-multiplayer-fps-maps-part-one
http://www.gamasutra.com/view/feature/195069/deathmatch_map_design_the_.php
https://www.youtube.com/watch?v=d4FSO2gtoXc
http://www.desiquintans.com/leveldesign
http://www.alteredgamer.com/game-development/60255-game-design-theory-multiplayer-level-design/
http://www.worldofleveldesign.com/categories/cat_level_design_tutorials_tips.php

**Comment by sinksinksink** [comment]
Sounds good. I'll check out the links later.

No need to make it overly complicated, especially as we haven't done any 3d modelling yet (suggest we do this at a point we can playtest and you can tweak the game while I do some 3d). If you need anything simple throwing together give me a shout.

**Comment by shauncg** [comment]
I've sketched out a couple of concepts and now just need to sit down and assemble them. Hopefully will at least make a start on that Thursday eve!

**Comment by shauncg** [comment]
map01_sg now has a really basic level within it for tinkering with. It's got a really rough interpretation of a layout I sketched out, with a bunch of elements missing because I didn't see the point in adding them yet. I've added physical boundary walls rather than invisible ones just because it seemed to make more sense with the end objective. 

I baked the lights but something weird is happening with the shadows. Not that it matters right now anyway. There are some other bugs too (object surfaces conflicting leading to flickering) but it'll do for now. 

The orthographic perspectives are really good for snapping things into place quickly.
